components/executors/k8s/patches/executor.ConfigMap.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: executor-config
+# Refer to https://docs.sourcegraph.com/admin/executors/deploy_executors_binary#step-2-setup-environment-variables on how to populate these variables
+data:
+  EXECUTOR_FRONTEND_URL: "http://sourcegraph.example.com"
```

Comment:
The executors should be able to talk to the frontend over the internal service declaration which keeps communications internal (and reduces bandwidth costs).